.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "singleQuote": true,
```

Comment:
Was committing this file intended? We currently don't have prettier in this repo.

Reply:
Hmm, this file does not affect anything indeed. Everything from [4efa828](https://github.com/payloadcms/payload/pull/1467/commits/4efa82820806b16ffa7ab2ba84535b5499132ad1) was me trying to get the intended formatting working.